packages/plugins/src/cli/setup.ts

```diff
@@ -65,58 +69,32 @@ export const makeCLI = (
           .hide('version'),
       async (argv) => proposeCommandHandler(argv, sphinxContext)
     )
-    // .command(
-    //   'artifacts',
-    //   `Retrieves deployment artifacts from the DevOps Platform and writes them to the file system.`,
-    //   (y) =>
-    //     y
-    //       .usage(
-    //         `Usage: sphinx artifacts --org-id <ORG_ID> --project-name <PROJECT_NAME>`
-    //       )
-    //       .option('org-id', {
-    //         describe: 'Your Sphinx organization ID.',
-    //         type: 'string',
-    //         demandOption: true,
-    //       })
-    //       .option('project-name', {
-    //         describe: 'The name of your project.',
-    //         type: 'string',
-    //         demandOption: true,
-    //       })
-    //       .hide('version')
-    //       .demandCommand(1, 'You must provide a Forge script path.'),
-    //   async (argv) => {
-    //     const { orgId, projectName } = argv
-
-    //     const apiKey = process.env.SPHINX_API_KEY
-    //     if (!apiKey) {
-    //       console.error(
-    //         "You must specify a 'SPHINX_API_KEY' environment variable."
-    //       )
-    //       process.exit(1)
-    //     }
-
-    //     const spinner = ora()
-    //     spinner.start(`Fetching artifacts...`)
-
-    //     const deploymentArtifacts = await fetchDeploymentArtifacts(
-    //       apiKey,
-    //       orgId,
-    //       projectName
-    //     )
-
-    //     spinner.succeed(`Fetched artifacts.`)
-    //     spinner.start(`Writing artifacts...`)
-
-    //     writeDeploymentArtifacts(
-    //       projectName,
-    //       ExecutionMode.Platform,
-    //       deploymentArtifacts
-    //     )
-
-    //     spinner.succeed(`Write artifacts.`)
-    //   }
-    // )
+    .command(
+      'artifacts',
+      `Retrieves deployment artifacts from the DevOps Platform and writes them to the file system.`,
+      (y) =>
+        y
+          .usage(
+            `Usage: sphinx artifacts --org-id <ORG_ID> --project-name <PROJECT_NAME>`
+          )
+          .option('org-id', {
+            describe: 'Your Sphinx organization ID.',
+            type: 'string',
+            demandOption: true,
+          })
+          .option('project-name', {
+            describe: 'The name of your project.',
+            type: 'string',
+            demandOption: true,
+          })
+          .option('silent', {
+            describe: 'Silence the output except for error messages.',
+            boolean: true,
+            default: false,
```

Comment:
If you want to add this as a user-facing flag, you should update the deployment artifact docs to include it. Also, I think you should update the usage section above, which is currently:
```
          .usage(
            `Usage: sphinx artifacts --org-id <ORG_ID> --project-name <PROJECT_NAME>`
          )
```